src/Shared/TypeExtensions.cs

```diff
@@ -1,10 +1,18 @@
 ﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
 // Licensed under the MIT License.  See License.txt in the project root for license information.
 
+using System.Reflection;
+
 namespace System
 {
     internal static partial class TypeExtensions
     {
+        private const BindingFlags QualifiedMethodBindingFlags = BindingFlags.NonPublic |
+                                                                 BindingFlags.Static |
```

Comment:
Actually, StyleCop has no preference on the indentation of initialization expressions. Just use this style to make it look prettier than the indentation as parameters.